tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -158,19 +161,24 @@ void toLaserScanRange(double x, double y, double z, double min_height,
 // included in laser scan.
 // @param max_height maximum height for a point of the point cloud to be
 // included in laser scan.
+// @param point_cloud_T_laser_message transform message between point cloud
+// frame and laser scan frame.
```

Comment:
More correct: Transformation from point cloud frame to laser scan frame. Or transform of laser scan wrt. point cloud.